src/main/java/exercise31/KarvonenHeartRate.java

```diff
@@ -13,19 +14,24 @@ public int formula(int age, int restingHR, double intensity) {
         return bpm;
     }
 
+    //Naming / data-type?
     public String[] checkDifferentIntensities(int age, int restingHR) {
+        //Shorter way to do addition?
         for (double intensity = 55; intensity <= 95; intensity = intensity + 5) {
+            //Brackets and whitespace?
             table[index] = (formula(age, restingHR, intensity) + "");
```

Comment:
The outer brackets are redundant.  Also the concatenation of the empty string to force the result into a `String` looks odd, and highlights the data-type issue already mentioned.